src/components/editor/GroupChoiceModal.jsx

```diff
@@ -28,18 +24,13 @@ const GroupChoiceModal = (props) => {
   }
 
   const saveAndClose = () => {
-    props.saveStarted()
-    const request = publishRDFResource(props.currentUser, props.rdf(), selectedValue)
+    props.publishMyResource(props.currentUser, selectedValue)
+
+    if (props.error) {
+      alert(props.error)
+      console.error(props.error)
+    }
 
-    request.then((result) => {
-      props.setBaseURL(result.response.headers.location)
-      // Need to regenerate RDF now that have baseURL
-      props.saveFinished(props.rdf())
-    }).catch((err) => {
-      alert('Unable to save resource')
-      console.error('unable to save resource')
-      console.error(err)
-    })
     props.closeRdfPreview()
     props.close()
```

Comment:
keeping these modal close actions here made sense to me, as they are pertinent to the logic of the GroupChoiceModal.  In fact, I originally closed them before showing the error message and an integration test failed.